core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -48,7 +49,7 @@ class LogManager(val logDirs: Array[File],
                  val retentionCheckMs: Long,
                  scheduler: Scheduler,
                  val brokerState: BrokerState,
-                 private val time: Time) extends Logging {
+                 time: Time) extends Logging {
```

Comment:
`private val` is not necessary here. A constructor without any qualifier in a plain class (i.e. not a case class) is not available outside the class body (i.e. constructor). And a field won't be created unless it's used outside the constructor (whereas `val` causes the field to be created, which is a bit odd if it's private, it's only needed in edge cases like if you want the field to be serialised or something like that).